addons/sourcemod/scripting/jwp_skin.sp

```diff
@@ -80,11 +81,11 @@ public Action Event_OnPlayerSpawn(Event event, const char[] name, bool dontBroad
 		TiB_SetSkin(client);
 		if (!g_bIsCSGO)
 		{
-			SetModel(client);
+			CreateTimer(0.5, SetModel, client);
```

Comment:
I think, it should be cvar'able value. It may cause some issues with vip plugins and other relevant stuff.